selene/schema/base.py

```diff
@@ -24,12 +24,23 @@
 from selene.schema.utils import get_text_from_element
 
 
-class BaseObjectType(graphene.ObjectType):
-    uuid = graphene.UUID(name='id')
-    name = graphene.String()
+class NameObjectTypeMixin:
+    name = graphene.String(description='Name of the object')
+
+    def resolve_name(root, _info):
+        return get_text_from_element(root, 'name')
+
+
+class UuidObjectTypeMixin:
```

Comment:
We had that discussion about `IP`.

what about
```suggestion
class UUIDObjectTypeMixin:
```
?